torch/testing/_internal/common_utils.py

```diff
@@ -1147,6 +1147,14 @@ def assertEqual(self, x, y, msg: Optional[str] = None, *,
                 assert debug_msg is not None
                 msg = "Scalars failed to compare as equal! " + debug_msg
             self.assertTrue(result, msg=msg)
+        # Tensor x Numpy array
```

Comment:
Should we also add a NumPy x NumPy case to be extra thorough?

Reply:
I added a case for Numpy x Numpy in the tests and it seems to be passing as is. Despite that, I could add it here too if you think it's a good idea

Reply:
Just so I can sleep easier at night. Let's just convert both to tensors.

Reply:
Done